content/en/docs/faq/_index.md

```diff
@@ -10,609 +10,9 @@ aliases: [
 # Frequently Asked Questions
 
 > What are the key differences between Helm 2 and Helm 3?  
```

Comment:
I suggest removing this line as well - it may confuse people since the scope of the FAQ goes beyond this single topic now.

Reply:
thanks @bridgetkromhout - removed that line